### PR TITLE
elly bootcamp code

### PR DESCRIPTION
**Pull request by elly-4**

### CODE REVIEW

**Comment by aydanjiwani** [comment]
Hey Elly, good start! For the bootcamp to count as complete, we require graphs for your NN accuracy and loss (I would recommend making these with matplotlib). Also please use pytorch for your ML library instead of tensorflow, because that's what our code currently uses. feel free to take a look at completed PRs or ping myself or Mika if you have any questions
